### ISSUE TITLE
Love your project

### ISSUE THREAD

**Issue by thelege2nd** [rafalgi/Timberman-Bot-Yolov8-object-detection#1]
Very good project, is it possible to also add mouse support and instead of using pyautogui, maybe something more lowerlevel so it could work for games? Most games wont notice payauto bcz they block virtual inputs, that's why lowerlevel is better.

**Comment by rafalgi** [comment]
Thank you for your feedback! I understand that you're suggesting adding mouse support and using a lower-level input method than pyautogui for game compatibility. This makes sense, as many games block virtual inputs from higher-level libraries like pyautogui. For game interactions, you'd want something that works at a lower level to bypass these restrictions. I can try to do this.

**Comment by thelege2nd** [comment]
Yes, exactly. The mouse should move to the object location detected by Yolo and aligns with it, then continues following the object if it moves, simultaneously at the same time, the keyboard can be utilized for responding to specific visual cues.   Waiting to see what you'll come up with :) You seem like a brilliant guy.
